packages/genesys-spark-components/src/components/stable/gux-form-field/components/gux-form-field-radio/gux-form-field-radio.scss

```diff
@@ -29,6 +29,7 @@
 
 :host {
   display: block;
+  padding-top: 4px;
```

Comment:
Does this need a token?

Reply:
There wasn't a token here, I moved this padding-top value from the `.gux-input-label` to `:host` so that we do not create a breaking change for existing users of this component. The new tokens for `padding-top` is  being applied to this component when it is part of the `gux-form-field-radio-group`: `var(--gse-ui-formControl-group-gapItems)`